.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: julia
+julia:
+  - 0.6
+  - nightly
```

Comment:
Do we really want nightly? Dev versions of Julia are pretty unstable in my experience.

Reply:
@ulysses4ever Let's remove it! I think what's more important is did try targeting [.0.4 and .0.5](https://travis-ci.org/pao/Monads.jl/builds/283498939), but they failed. I personally don't mind nightly testing, but trying builds on release versions should definitely be done.